fdbclient/NativeAPI.actor.cpp

```diff
@@ -4109,6 +4110,7 @@ ACTOR Future<Version> extractReadVersion(Location location, SpanID spanContext,
 	}
 
 	metadataVersion.send(rep.metadataVersion);
+	span.addTag(LiteralStringRef("read_version"), std::to_string(rep.version));
```

Comment:
This is a memory bug - the string returned by `to_string` is a temporary. The proper solution is to define a `Standalone<StringRef>` that has a longer lifetime than `span` (a state variable after the `Span` should do the trick).

Reply:
Something like this?

```
state Standalone<StringRef> version(std::to_string(rep.version));
```

This still seems non-ideal because it creates a string and then copies the data into another string in the `StringRef` constructor.

Reply:
I removed this tag so this point is moot :)